wp-content/themes/csisnuclearnetwork/assets/_scss/abstracts/_mixins.scss

```diff
@@ -131,6 +131,27 @@
   }
 }
 
+@mixin full-width-bg-2(
+  $background,
+  $direction: '',
+  $includeMargins: true
+) {
+  $width_forumula: calc((100vw - 100%) / 2);
+  position: relative;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: calc(-1 * calc((100vw - 100%) / 2));
+    z-index: -2;
+    display: block;
+    width: calc(100% + calc(100vw - 100%));
+    height: 100%;
+    background: $background;
+  }
+}
+
```

Comment:
Since this is only used on the footer, let's not use a mixin and move the `before` code to the footer file instead.